### ISSUE TITLE
Extra Slot Save path load.

### ISSUE THREAD

**Issue by FakelsHub** [phobos2077/sfall#136]
Если в папке игры есть две разные папки data, и в конфиге fallout2.cfg путь master_patches= будет указывать на другую папку, то сохранения будут видны и одновременно из двух папок. :)

**Comment by FakelsHub** [comment]
https://github.com/phobos2077/sfall/blob/204810d0670f5ad56fe9735055eca9ee64a344b7/sfall/SuperSave.cpp#L31
This file not create in 'savegame' folder.

**Comment by NovaRain** [comment]
The file is created on my test machines. Or do you mean it won't be created if master_patches and critter_patches are pointing to two different paths?

**Comment by FakelsHub** [comment]
> Or do you mean it won't be created if master_patches and critter_patches are pointing to two different paths?

No. The file is not created at all, it can not be read if it is created.
Game location: F:\Fallout2\Data
master_patches=data
System: WinXP

**Comment by NovaRain** [comment]
>No. The file is not created at all, it can not be read if it is created.

If you're using 4.0.3/3.8.8, it's perfectly normal. Because it's been broken lone time ago, only fixed in the upcoming 4.0.4/3.8.9.

**Comment by FakelsHub** [comment]
> Because it's been broken lone time ago, only fixed in the upcoming 4.0.4/3.8.9.

Damn, I thought something was corrected for 3.8.8 :)

**Comment by NovaRain** [comment]
Enabling DataLoadOrderPatch in ddraw.ini should solve this problem. I'm not sure if it's better to make it enabled by default or simply force-enabled it.

**Comment by FakelsHub** [comment]
By the way, the same bug affects other features. for example, loading sfall scripts from the data folder.
It seems that the data folder is dead sewn somewhere in the engine.

**Comment by NovaRain** [comment]
> By the way, the same bug affects other features. for example, loading sfall scripts from the data folder.

You mean if master_patches & critter_patches are set to different folders, and they both have their own "scripts" folder? And you put global/hook scripts in them? IMO setting the two to different folders are something that shouldn't happen unless being done intentionally (by mod packer or player). Maybe add a hack or something to force unify the two on start?

**Comment by FakelsHub** [comment]
Example: master_patches=data2, critter_patches=data2
The `data` folder is still active. this is the crux of the problem.

**Comment by FakelsHub** [comment]
ha. I just found the reason for this: f2_res_patches=data\
now it is clear where this folder is attached from.
